frontend/src/types/sql-review-schema.yaml

```diff
@@ -12,20 +12,23 @@ ruleList:
     engineList:
       - MYSQL
     componentList: []
+    availablePlanList: [1, 2]
```

Comment:
Shall we build a map in Go code because this is only for Bytebase console?

Reply:
Synced with @tianzhou, we only need to add this limit in the frontend, and don’t need to check the permission in the backend, so I prefer to put them into the frontend files

Reply:
I see, shall we do it in the frontend console code? Enterprise code comes together, giving us a chance to provide more complex enterprise plans later.

Reply:
Updated, PTAL